blockchain/pkg/ddcprimitives/ddcprimitives.go

```diff
@@ -0,0 +1,74 @@
+/*
+Primitive types definition of the Cere Network blockchain.
+
+Find more about SCALE encoding/decoding for enums here: https://github.com/centrifuge/go-substrate-rpc-client/blob/0a28e8b/types/example_enum_test.go#L27.
+
+Based on [ddc-primitives@0.1.0].
+
+[ddc-primitives@0.1.0]: https://github.com/Cerebellum-Network/blockchain-node/blob/4631f58/primitives/src/lib.rs#L20.
+*/
+package ddcprimitives
```

Comment:
filename duplicates package (directory) name. might be it makes sense to remove directory?

Reply:
I would agree because a single file package usually makes no sense. But here we have definitions used in `pallets` sub-package and if we put `ddcprimitives.go` in the `pkg` it creates an import cycle (`pkg/client.go` imports `pkg/pallets` for pallets APIs and `pkg/pallets` import `pkg` for primitive types). Looks like we can put everything in the `pkg` (or even remove `pkg` and put everything in the `blockchain`) removing `pallets` package or leave it as it is. What do you prefer?

Reply:
might be it make sense to put primitives.go next to cluster.go and customers.go? pallets folder can be kept

Reply:
Moved here: https://github.com/Cerebellum-Network/cere-ddc-sdk-go/pull/70/commits/1c6a122a80fe7bb59c43503f74b0e75d9ce480b4.

But now an import looks like
`import "github.com/cerebellum-network/cere-ddc-sdk-go/blockchain/pallets"`

and an item usage is
`pallets.NodePubKey`.

Before we had
`import "github.com/cerebellum-network/cere-ddc-sdk-go/blockchain/ddcprimitives"`

and an item usage was
`ddcprimitives.NodePubKey`

which looks better to me.

Please let me know if it's ok or I better revert the patch.